### DISCUSSION TITLE
Merging new processors

### DISCUSSION THREAD

**Discussion by Tim---** [NationalSecurityAgency/ghidra#2602] [General]
Hi !

I have been working on adding a processor in adding a processor for Ghidra, and my PR was greeted with a [Feature: Processor/Unsupported](https://github.com/NationalSecurityAgency/ghidra/pulls?q=is%3Apr+is%3Aopen+label%3A%22Feature%3A+Processor%2FUnsupported%22) tag (oh noes !). 
I looked at some of the PRs and I didn't see a discussion about this, so I'm starting it here.

I guess the reason is that if you support new processors, this means that you get more issues/pull requests. Given that the project is already really busy, I understand that you must cherry pick the features that you want to add.

So this brings me two questions:

- Where should we put the code for new processors ? Should we create an OOT repository, or can we keep a long-running PR open in this repo ?
- Also, do you have an agenda or general requirements for a processor to be added to Ghidra ?

**Comment by ryanmkurtz** [comment]
The intent of the `Feature: Processor/Unsupported` label is just to categorize all the of the processors that are not currently merged into Ghidra.  It was never intended to mean the PR would never be accepted.  Perhaps a better name for the label would be `Feature: Processor/New` or `Feature: Processor/Candidate`.
